README.md

```diff
@@ -183,6 +183,20 @@ There is support for SI units through the use of `si_unit`. Units declared throu
   * ft, foot, feet
   * yd, yard, yards
   * mi, mile, miles
+* `Measured::Volume`
+  * l, liter, litre, liters, litres
+  * m3, cubic_meter, cubic_meters, cubic_metre, cubic_metres
+  * ml, milliliter, millilitre, milliliters, millilitres
```

Comment:
If you specify that litres covers all SI prefixes you won't need this separate README line